devel/revive-hf.lua

```diff
@@ -0,0 +1,40 @@
+-- Revives the specified historical figure
+
+--[====[
+
+devel/revive-hf
+=============
+
+Revivs the specified historical figure, even if off-site.
+
+Usage::
+
+    devel/revive-hf HISTFIG_ID
+
+Arguments:
+
+``histfig_id``:
+    the ID of the historical figure to target
+
+]====]
```

Comment:
Note that documentation should now go in the `docs/` subdirectory. If we keep this as a separate file, you can (should) remove the doc comment inside the script file.